tests/pydantic_v1_parsing_test.py

```diff
@@ -0,0 +1,130 @@
+from datetime import datetime, timezone
+from typing import Union
+from zoneinfo import ZoneInfo
+
+import pytest
+from parameterized import parameterized  # type: ignore[import-untyped]
+from pydantic.v1 import BaseModel, ValidationError
```

Comment:
this file is the same as `tests/pydantic_parsing_test.py` except for importing `pydantic.v1` instead of `pydantic` (and using `parse_obj` instead of `model_validate`